extensions/notebook/src/book/bookModel.ts

```diff
@@ -167,12 +170,12 @@ export class BookModel implements azdata.nb.NavigationProvider {
 						);
 						notebooks.push(markdown);
 					} else {
-						let error = loc.missingFileError(sections[i].title);
-						vscode.window.showErrorMessage(error);
+						this._errorMessage = loc.missingFileError(sections[i].title);
+						vscode.window.showErrorMessage(this._errorMessage);
 					}
 				}
 			} else {
-				// TODO: search functionality (#6160)
+				// toDO: search functionality (#6160)
```

Comment:
Why are you changing the casing for a bunch of these comments? That doesn't seem necessary.

Reply:
the ts lint erros started popping up and the amber lines are not fin while am making changes, have changed some to avoid them :|